model_card_toolkit/tfx/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""
+Removal notice for the ModelCardGenerator TFX component.
+"""
+
+raise NotImplementedError(
```

Comment:
I have never seen this pattern before to raise the error directly in the `__init__.py`. Seems pretty cool to do it this way.

Reply:
Changed to `ImportError`. 😅